arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,1020 @@
+			"EN_OFF_BD_5V",
+			"EN_CELL_MODEM_PWR",
+			"EN_NIMBEL_3.3V",
+			"EN_GPS_PWR#",
+			"EN_CAN_XVR#",
+			"EN_232_XVR",
+			"EN_LS_OUT_1",
+			"EN_LS_OUT_2",
+			"EN_LS_OUT_3",
+			"EN_LS_OUT_4",
+			"EN_LS_OUT_5",
+			"EN_LS_OUT_6",
+			"EN_LS_OUT_7",
+			"NA",
+			"NA",
+			"NA",
+			/* Inputs 32-63 */
+			"DIG_IN_1",
+			"DIG_IN_2",
+			"DIG_IN_3",
+			"DIG_IN_4",
+			"SD_BOOT_JMP#",
+			"DIO_1_IN",
+			"DIO_2_IN",
+			"DIO_3_IN",
+			"DIO_4_IN",
+			"DIO_5_IN",
+			"DIO_6_IN",
+			"DIO_7_IN",
+			"NA",
+			"NA",
+			"NA",
+			"N7",
+			"P7",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA",
+			"NA";
+	};
+};
+
+&snvs_poweroff {
+	status = "okay";
+};
+
+&snvs_rtc {
+	status = "disabled";
+};
+
+&wdog1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_wdog>;
+	fsl,wdog_b;
+	status = "disabled";
+};
+
+&pwm5 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_pwm5>;
+	status = "okay";
+};
+
+&iomuxc {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_hog>;
+	imx6ul-ts7180 {
+		pinctrl_hog: hoggrp {
+			fsl,pins = <
+				MX6UL_PAD_UART1_CTS_B__GPIO1_IO18 	0x1b020 /* SPARE_1 */
+				MX6UL_PAD_SNVS_TAMPER0__GPIO5_IO00	0x10020 /* POWER_FAIL */
+				MX6UL_PAD_SNVS_TAMPER5__GPIO5_IO05	0x10020 /* SILAB_C2_CLK */
+				MX6UL_PAD_SNVS_TAMPER6__GPIO5_IO06	0x10020 /* SILAB_C2_DAT */
+				MX6UL_PAD_SNVS_TAMPER7__GPIO5_IO07	0x10020 /* SILAB_C2_RESET */
+				MX6UL_PAD_SNVS_TAMPER2__GPIO5_IO02	0x1b020 /* EN_FPGA_PWR on REV C and below */
+				MX6UL_PAD_LCD_DATA05__GPIO3_IO10	0x10020 /* HD12_DIG_INPUT */
+				MX6UL_PAD_LCD_DATA06__GPIO3_IO11	0x10020 /* NO_CHRG_JMP# */
+				MX6UL_PAD_LCD_DATA07__GPIO3_IO12	0x10020 /* EN_NIM_USB# */
+				MX6UL_PAD_LCD_DATA11__GPIO3_IO16	0x10020 /* U_BOOT_JMP# */
+				MX6UL_PAD_LCD_DATA13__GPIO3_IO18	0x10000 /* PUSH_SW_CPU# */
+				MX6UL_PAD_LCD_DATA14__GPIO3_IO19	0x1b020 /* NIMBEL_PWR_ON */
+				MX6UL_PAD_LCD_DATA18__GPIO3_IO23	0x1b020 /* ID4 */
+				MX6UL_PAD_LCD_DATA22__GPIO3_IO27	0x1b020 /* ID1 */
+				MX6UL_PAD_NAND_ALE__GPIO4_IO10  	0x03020 /* MAG_N_IRQ */
+				MX6UL_PAD_NAND_WP_B__GPIO4_IO11		0x1b020 /* FPGA_RESET# */
+				MX6UL_PAD_LCD_DATA19__GPIO3_IO24	0x1b020 /* JTAG_FPGA_TDO */
+				MX6UL_PAD_LCD_VSYNC__GPIO3_IO03		0x1b020 /* JTAG_FPGA_TDI */
+				MX6UL_PAD_LCD_HSYNC__GPIO3_IO02		0x1b020 /* JTAG_FPGA_TMS */
+				MX6UL_PAD_LCD_ENABLE__GPIO3_IO01	0x1b020 /* JTAG_FPGA_TCK */
+				MX6UL_PAD_CSI_DATA00__GPIO4_IO21	0x10020 /* 6UL_FORCE_5V_ON */
+				MX6UL_PAD_CSI_DATA02__GPIO4_IO23	0x1b020 /* EN_YEL_LED# */
+				MX6UL_PAD_CSI_DATA03__GPIO4_IO24	0x1b020 /* EN_RED_LED# */
+				MX6UL_PAD_CSI_DATA04__GPIO4_IO25	0x1b020 /* EN_GREEN_LED# */
+				MX6UL_PAD_CSI_DATA05__GPIO4_IO26	0x1b020 /* EN_BLUE_LED */
+				MX6UL_PAD_LCD_DATA06__GPIO3_IO11	0x1b020 /* NO_CHRG_JMP# */
+				MX6UL_PAD_CSI_VSYNC__GPIO4_IO19		0x1b020 /* ID5 */
+				MX6UL_PAD_CSI_HSYNC__GPIO4_IO20		0x03020 /* GYRO_INT */
+			>;
+		};
+
+		pinctrl_ppsgpio: ppsgpio {
```

Comment:
All pinctrl blocks should be in the format of `pinctrl_<useful label>: <useful node>grp{`